src/DependencyInjection/WebgriffeSyliusAkeneoExtension.php

```diff
@@ -99,6 +100,13 @@ final class WebgriffeSyliusAkeneoExtension extends AbstractResourceExtension imp
                 'filesystem',
             ],
         ],
+        'metric_property' => [
+            'class' => MetricPropertyValueHandler::class,
+            'arguments' => [
+                'property_accessor',
+                'webgriffe_sylius_akeneo.converter.default_unit_measurement_value_converter'
+            ],
+        ],
```

Comment:
This public static array is deprecated and must not be used changed for BC (see [BC check](https://github.com/webgriffe/SyliusAkeneoPlugin/pull/83/checks?check_run_id=2554859693)).

```suggestion
```